README.md

```diff
@@ -1,118 +1,171 @@
-ZendSkeletonApplication
-=======================
+# ZendSkeletonApplication
 
-Introduction
-------------
-This is a simple, skeleton application using the ZF2 MVC layer and module
+## Introduction
+
+This is a skeleton application using the Zend Framework MVC layer and module
 systems. This application is meant to be used as a starting place for those
-looking to get their feet wet with ZF2.
+looking to get their feet wet with Zend Framework.
 
-Installation using Composer
----------------------------
+## Installation using Composer
 
-The easiest way to create a new ZF2 project is to use [Composer](https://getcomposer.org/). If you don't have it already installed, then please install as per the [documentation](https://getcomposer.org/doc/00-intro.md).
+The easiest way to create a new Zend Framework project is to use
+[Composer](https://getcomposer.org/).  If you don't have it already installed,
+then please install as per the [documentation](https://getcomposer.org/doc/00-intro.md).
 
+To create your new Zend Framework project:
 
-Create your new ZF2 project:
+```bash
+$ composer create-project -n -sdev zendframework/skeleton-application path/to/install
+```
 
-    composer create-project -n -sdev zendframework/skeleton-application path/to/install
+Once installed, you can test it out immediately using PHP's built-in web server:
 
+```bash
+$ php -S 0.0.0.0:8080 -t public/ public/index.php
+```
 
+This will start the cli-server on port 8080, and bind it to all network
+interfaces.
 
-### Installation using a tarball with a local Composer
+**Note:** The built-in CLI server is *for development only*.
 
-If you don't have composer installed globally then another way to create a new ZF2 project is to download the tarball and install it:
+## Using Vagrant
 
-1. Download the [tarball](https://github.com/zendframework/ZendSkeletonApplication/tarball/master), extract it and then install the dependencies with a locally installed Composer:
+This skeleton includes a `Vagrantfile` based on ubuntu 14.04, and using the
```

Comment:
Nope, tried it. VirtualBox 5.0.18 has a known bug when running `ubuntu/xenial64` (I actually ran into it today when testing). We can update later once the issues are sorted.